app/models/requisition.rb

```diff
@@ -1,4 +1,4 @@
-class ApplicantsPosition < ApplicationRecord
+class Requisition < ApplicationRecord
   belongs_to :applicant
   belongs_to :position
   enum status: [:applicant_interested, :currently_interviewing, :intend_to_hire]
```

Comment:
We just talked, and agreed to make the statuses "hire" and "do_not_hire" - please use these words or let me know if they change.